docs/use_case/postv1.md

```diff
@@ -0,0 +1,77 @@
+This initial version of the demo #post app will allow you to create and visualise your own post(s). It will also showcase examples of the ONEM's python SDK usage.
+
+So let's navigate to the **postv1** folder and create/activate our virtual environment:
+
+```bash
+onem@local:~$ cd ~/postv1 ; pipenv shell --python 3.7
+```
+
+
+Also please make sure that:
+
+ - the database is created (even if we will store the posts in-memory).
+ - local django server is started.
+ - the server is exposed publicly.
+ - the application is registered.
+All of the above steps are explained in the [run locally](/getting_started/python_run_local/) section of the docs.
+
+
+To register your application please open [ONEm Developer Portal]({{ links.portal }}) and register for a free account.
```

Comment:
Lines 19 up to 23 should be covered by the running locally tutorial

Reply:
They do, but @chrishornmem asked for these to be mentioned again here. Check previous comments.